src/borg_import/main.py

```diff
@@ -37,11 +37,20 @@ def borg_import(args, archive_name, path, timestamp=None):
 
 def list_borg_archives(args):
     borg_cmdline = ['borg', 'list', '--short']
-    repository = args.repository.resolve()
+    repository = repo_resolve(args.repository)
```

Comment:
I  had a another look:

The root cause is that args.repository is a Path object (see argparse setup), but that isn't true in any case and this is what's causing the issue here. The repo url given on the command line should not become a Path object in the first place.